.github/workflows/lint.yml

```diff
@@ -19,17 +19,27 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
-      - name: Install tox
-        run: pip install "tox < 4"
+
+      - name: install packages to tests
+        run: sudo pip install ./psycopg[dev,test] codespell
```

Comment:
It'd be nice to also drop the `sudo pip install` in this file.

Reply:
Yes, done. It was needed because I forgot the
```
with:
    python-version: something
```
in the `setup-python` action.